behave/features/steps/cla_contact_us.py

```diff
@@ -18,14 +18,43 @@ def step_impl_means_test(context):
 
 
 @step(u'I click \'continue to contact CLA\'')
-def step_click_submit(context):
+def step_impl(context):
+    context.execute_steps(u'''
+        Given I click continue
+    ''')
+
+
+@step(u'I select \'Submit details\'')
+def step_impl(context):
     context.execute_steps(u'''
         Given I click continue
     ''')
 
 
-@step(u'I am taken to the \'contact civil legal advice\' page')
+@step(u'I on to the \'contact civil legal advice\' page')
```

Comment:
Missing a word :) "I am on the ..."

Reply:
Actually can we change this so it uses the generic step `And I am taken to the "{header}" page located on "{path}"`?